### PR TITLE
User crud 

### PR DESCRIPTION
**Pull request by jitendrasbunde**

### CODE REVIEW

File: userCrud/config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "domain": "www.example.com",
+  "mongodb": {
+    "username": "jitendra",
+    "password": "jitu2121"
+  }
+}
```
Comment (sunil4sonawane) [inline userCrud/config.json]: new line required

File: userCrud/userApi.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      console.log(err);
+    }
+    console.log("1 document inserted");
+
+  });
+
+  res.json({'url':'`json object sended`'});
+});
+
+module.exports = router;
```
Comment (sunil4sonawane) [inline userCrud/userApi.js]: new line required

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
Check comments

File: userCrud/config.json (hunk 1 above)
Comment (sunil4sonawane) [reply userCrud/config.json]: While using env params, no need of config

Comment (jitendrasbunde) [reply userCrud/config.json]: ok sir

**Pull request merged**